src/data_management/scenarios/Scenarios.js

```diff
@@ -24,10 +24,7 @@ class Scenarios extends Component {
     };
 
     fetchScenariosFromApi = (organisationUUID, page, searchContains, ordering) => {
-        //Use short UUID of organisation instead of the full UUID due to backend technical problem
-        //This should be fixed in the future
-        const shortUUID = organisationUUID.substr(0, 7);
-        const url = `/api/v4/scenarios/?writable=true&page_size=${this.state.pageSize}&organisation__uuid=${shortUUID}&page=${page}&name__icontains=${searchContains}&ordering=${ordering}`;
+        const url = `/api/v4/scenarios/?writable=true&page_size=${this.state.pageSize}&organisation__uuid=${organisationUUID}&page=${page}&name__icontains=${searchContains}&ordering=${ordering}`;
```

Comment:
Since the backend filtering on organisation uuid is now working properly with full uuid of the organisation, I switch this back to use the full uuid instead of short uuid. Everything still works the same.